packages/ckeditor5-table/src/tablecolumnresize/tablecolumnresizeediting.js

```diff
@@ -730,3 +748,16 @@ export default class TableColumnResizeEditing extends Plugin {
 		}, { priority: 'lowest' } );
 	}
 }
+
+// Helper function for checking if the table is:
+// 	- aligned right with left-to-right content language direction
+// 	- aligned left with right-to-left content language direction
+//
+// The function got extracted to avoid code duplication as we are using it multiple times.
```

Comment:
Good docs except this very line.

This should be obvious for everyone, no need to mention detail like that.

Also add API docs for params, returned value.

As an example see:

https://github.com/ckeditor/ckeditor5/blob/aa0d99bd17c066d73518927157d7c4c7cfeda4be/packages/ckeditor5-list/src/listproperties/listpropertiesediting.js#L817-L818